packages/eslint-config-khan/index.js

```diff
@@ -4,9 +4,14 @@ const WARN = "warn";
 const ERROR = "error";
 
 module.exports = {
-    parser: "babel-eslint",
-    plugins: ["jsx-a11y", "prettier", "react"],
-    extends: ["eslint:recommended", "prettier"],
+    parser: "@typescript-eslint/parser",
+    plugins: ["@typescript-eslint", "jsx-a11y", "prettier", "react"],
```

Comment:
Will these changes impact other non-TS packages that are importing eslint-config?

Reply:
It will, but only if they update to the new version.  It looks like wonder-block, perseus, and mobile/webview are using it.

Reply:
I'll be cleaning this up in the next PR since lots of things that should be in this file are actually in .eslintrc.js right now.